test/Microsoft.AspNet.Facebook.Test/Microsoft.AspNet.Facebook.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\..\packages\xunit.runner.visualstudio.2.3.1\build\net20\xunit.runner.visualstudio.props" Condition="Exists('..\..\packages\xunit.runner.visualstudio.2.3.1\build\net20\xunit.runner.visualstudio.props')" />
-  <Import Project="..\..\packages\xunit.core.2.3.0\build\xunit.core.props" Condition="Exists('..\..\packages\xunit.core.2.3.0\build\xunit.core.props')" />
+  <Import Project="..\..\packages\xunit.runner.visualstudio.2.4.5\build\net462\xunit.runner.visualstudio.props" Condition="Exists('..\..\packages\xunit.runner.visualstudio.2.4.5\build\net462\xunit.runner.visualstudio.props')" />
```

Comment:
Would it be straightforward to parameterize versions in this repo? Would make future changes easier. But if it's pretty involved then it's probably not worth it.

Reply:
It would be fairly involved because packages.config files can't use `msbuild` variables. Therefore, not worthwhile.

In addition, it's worth doing these changes manually (well, w/ a careful regular expression or two 😉) because it gives us an opportunity to check the assembly paths (in case new assemblies are available).